src/types/types.ts

```diff
@@ -21,7 +21,7 @@ export type UserDataType = {
     lastName: string;
     displayName: string;
     location: string;
-    avatar: string;
+    avatarUrl: string;
```

Comment:
This is a fix that should be in PR #1 , but was missed. The API data returns `avatarUrl `instead of `avatar` (Although the links that come from the API are all unavailable and the avatar images uses the `alt `property because of that)